protobuf/treasury_contract.proto

```diff
@@ -26,9 +26,10 @@ service TreasuryContract {
     }
     rpc ChangeVoteWeightInterestController (acs1.AuthorityInfo) returns (google.protobuf.Empty) {
     }
+    rpc SetAmountAndTimeProportionOfVoteWeight (AmountAndTimeProportion) returns (google.protobuf.Empty) {
```

Comment:
Provide vote weight related information in Election Contract, do not add same interfaces here.